etc/postgres_connect_check/pach_postgres_check.go

```diff
@@ -0,0 +1,36 @@
+// pach_postgres_check.go
+
+package main
+
+import (
+    "context"
+    "fmt"
+    "os"
+    "flag"
+    "time"
+
+    "github.com/jackc/pgx/v5"
+)
+
+func main() {
+
+    connectionString := flag.String("connection_string", "", "postgres database connection string")
+    timeout := flag.Float64("timeout", 10, "connection attempt timeout in seconds")
```

Comment:
This should be a flag.Duration.  This will allow the flag to be in this format:

```
    ParseDuration parses a duration string. A duration string is a possibly
    signed sequence of decimal numbers, each with optional fraction and a unit
    suffix, such as "300ms", "-1.5h" or "2h45m". Valid time units are "ns",
    "us" (or "µs"), "ms", "s", "m", "h".
```